UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.ts

```diff
@@ -549,9 +554,33 @@ export class MangaReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         takeUntilDestroyed(this.destroyRef)
       ).subscribe(() => {});
 
+      this.generalSettingsForm.get('pageSplitOption')?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+        const fitting = this.generalSettingsForm.get('fittingOption')?.value;
+        if(PageSplitOption.FitSplit == val && FITTING_OPTION.WIDTH == fitting) {
```

Comment:
Let's add a bit of documentation about our decisions about when this should work. It will likely help me in the future when there is a bug.